### PR TITLE
split telemetry table into three tables and addition more columns int…

### PR DESCRIPTION
**Pull request by devashishRaj**
Split telemetry.rpi_metrics into three tables  , updated relevant section of postgres.md

telemetry.devices , telemetry.rpi4b_metrics , telemetry.rpi_temp_alert

added more data : hostname , ostype , private and public ip  in telemetry.devices.

made timestamp column unique

### CODE REVIEW

**Comment by devashishRaj** [comment]
@agaurav 
https://github.com/devashishRaj/rpi_telemetry/discussions/2

File: client/scraprpi/scraprpi.go [hunk 1]
```diff
@@ -14,89 +13,115 @@ import (
...
+		log.Fatalln(err)
 	}
 }
 
 type SystemInfo struct {
-	HardwareID  string  `json:"HardwareID"`
+	MacAddress  string  `json:"MacAddress:"`
 	CPUuserLoad float64 `json:"CPUuserLoad"`
-	CPUidle     float64 `json:"CPUidle"`
 	TotalMemory int64   `json:"TotalMemory"`
```
Comment (agaurav) [inline client/scraprpi/scraprpi.go:26]: this struct also needs to be split into two structs, one for `info` other for `metric`

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    image: postgres:latest
+    container_name: postgres
+    environment:
+      POSTGRES_DB: rospi
+      POSTGRES_USER: rospi
+      POSTGRES_PASSWORD: rpitele
+    ports:
+      - "5432:5432"
+    volumes:
+      - /Users/devashishraj/Desktop/postgresGO/raspi_IOT/rpi_telemetery/schema.sql:/docker-entrypoint-initdb.d/schema.sql
```
Comment (agaurav) [inline docker-compose.yml:13]: use `$HOME` or relative path here. others systems are not going to have the same dir structure.

File: server/dataStruct/struct.go [hunk 3]
```diff
@@ -1,12 +1,15 @@
 package datastruct
 
 type SystemInfo struct {
```
Comment (agaurav) [inline server/dataStruct/struct.go:3]: this struct is being duplicated across client and server code. 
client should import it from a server package for use.

File: client/go.mod [hunk 4]
```diff
@@ -4,4 +4,23 @@ go 1.21.0
 
 require github.com/mackerelio/go-osstat v0.2.4
```
Comment (agaurav) [inline client/go.mod:5]: ideally server and client should have same `go.mod` to ensure they are using same version of go packages for compatibility.

**Review by agaurav — APPROVED** [review]
good progress.
This can be merged with the suggested changes.

**Comment by devashishRaj** [comment]
will create separate branch to have for one go mod and single data struct .

**Pull request merged**